hal/src/sercom/v2/i2c/pads_thumbv6m.rs

```diff
@@ -0,0 +1,163 @@
+//! Define a container for a set of SERCOM pads
+//!
+//! See the [i2c module](super) documentation for more details on declaring and
+//! instantiating a [`Pads`] type.
+
+use crate::{gpio::v2::AnyPin, sercom::v2::*, typelevel::Sealed};
+use core::marker::PhantomData;
+
+/// Container for a set of SERCOM [`Pad`]s
+///
+/// See the [module-level](crate::sercom::v2::i2c) documentation for more
+/// details on specifying a `Pads` type and creating instances.
+pub struct Pads<S, SDA, SCL>
+where
+    S: Sercom,
+    SDA: IsI2cPad<PadNum = Pad0, Sercom = S>,
+    SCL: IsI2cPad<PadNum = Pad1, Sercom = S>,
+{
+    sercom: PhantomData<S>,
+    sda: SDA,
+    scl: SCL,
+}
+
+/// Container for a set of SERCOM [`Pad`]s
+///
+/// See the [module-level](crate::sercom::v2::i2c) documentation for more
+/// details on specifying a `Pads` type and creating instances.
+#[cfg(feature = "samd11")]
+impl<S, DI, CI> Pads<S, Pad<S, Pad0, DI>, Pad<S, Pad1, CI>>
```

Comment:
Same comment about `PadsFromIds`

Reply:
I did the same for the samd21 impl, but `PadsFromIds` doesn't exist for samd11 targets.

Reply:
I think it actually could for I2C. The problem with SAMD11 is that knowing the `Sercom` and the `PinId` is not enough, because there is ambiguity in the `PadNum`. But for I2C, you know the `PadNum` for both pins. I think you could do

```rust
pub type PadsFromIds<S, SDA, SCL> = Pads<S, Pad<S, Pad0, SDA>, Pad<S, Pad1, SCL>>;
```

Reply:
Huh indeed you're right. Just implemented it.